<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjectDialog</class>
 <widget class="QDialog" name="AddObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="objectNameLb">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="hideCheckBox">
       <property name="text">
        <string>Hide to all users</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1" rowspan="2">
      <widget class="QCheckBox" name="lockCheckBox">
       <property name="text">
        <string>Lock to others</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="objectWidthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>px</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="objectHeightSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>px</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>HTML Text:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="0" column="2" rowspan="12" colspan="2">
      <widget class="QGraphicsView" name="previewGraphicsView"/>
     </item>
     <item row="12" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>600</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="3">
      <widget class="QPushButton" name="addImageBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add Item</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/add</normaloff>:/images/add</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Create a new item based on an image and add it to the table.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addAllBtn">
       <property name="text">
        <string>Add All from Table File</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/box</normaloff>:/images/box</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseImageBtn">
       <property name="text">
        <string>Browse Image</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/find</normaloff>:/images/find</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browseImageBtn</tabstop>
  <tabstop>addAllBtn</tabstop>
  <tabstop>objectWidthSpinBox</tabstop>
  <tabstop>objectHeightSpinBox</tabstop>
  <tabstop>hideCheckBox</tabstop>
  <tabstop>lockCheckBox</tabstop>
  <tabstop>addImageBtn</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>previewGraphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Engine/EResources.qrc"/>
 </resources>
 <connections/>
</ui>
